<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Push</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="itemToPushedLineEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="popButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Pop</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="topLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TOP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="item2">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>267</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item3">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>239</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item7">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>127</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item8">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>99</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item1">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>295</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item4">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>211</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item6">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>155</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item5">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>183</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item9">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>71</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="item0">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>323</y>
      <width>59</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stack9">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>71</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[9]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack8">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>100</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[8]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack7">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>128</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[7]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack6">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>157</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[6]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack5">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>185</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[5]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack4">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>214</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[4]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack3">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>242</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[3]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack2">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>271</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[2]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack1">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>299</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[1]</string>
    </property>
   </widget>
   <widget class="QLabel" name="stack0">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>328</y>
      <width>26</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S[0]</string>
    </property>
   </widget>
   <zorder>stack9</zorder>
   <zorder>stack8</zorder>
   <zorder>stack7</zorder>
   <zorder>stack6</zorder>
   <zorder>stack5</zorder>
   <zorder>stack4</zorder>
   <zorder>stack3</zorder>
   <zorder>stack2</zorder>
   <zorder>stack1</zorder>
   <zorder>stack0</zorder>
   <zorder>item2</zorder>
   <zorder>item3</zorder>
   <zorder>item7</zorder>
   <zorder>item8</zorder>
   <zorder>item1</zorder>
   <zorder>item4</zorder>
   <zorder>item6</zorder>
   <zorder>item5</zorder>
   <zorder>item9</zorder>
   <zorder>item0</zorder>
   <zorder>pushButton</zorder>
   <zorder>itemToPushedLineEdit</zorder>
   <zorder>popButton</zorder>
   <zorder>topLabel</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
